scripts/ErrorReporter/errorreport.py

```diff
@@ -15,21 +15,39 @@
 else:
     raise RuntimeError("Unknown QT version: {}".format(qtpy.QT_VERSION))
 
-from qtpy import QtCore, QtGui, QtWidgets # noqa: E402
-from qtpy.QtCore import Signal # noqa: E402
-from qtpy.QtWidgets import QMessageBox # noqa: E402
-from mantidqt.utils.qt import load_ui # noqa: E402
+from qtpy import QtCore, QtGui, QtWidgets  # noqa: E402
+from qtpy.QtCore import Signal, QUrl  # noqa: E402
+from qtpy.QtGui import QDesktopServices  # noqa: E402
+from qtpy.QtWidgets import QMessageBox, QTextEdit  # noqa: E402
```

Comment:
QTextEdit is unused here as well I think. 

Flake8 should normally pick up on any unused imports but the `# noqa` comments are telling it to ignore those lines. I know this wasn't you, but I think it would be better to have the `noqa` comments on lines 11-16 rather than on every import, then we're still warned if any are unused